notewhacker.lisp

```diff
@@ -96,7 +96,17 @@ element."
     :accessor midi-channel :initarg :midi-channel :initform nil
     :documentation "Events with matching channel information target
     this staff. If nil, all events match. Used for matching Midi
-    events to staff."))
+    events to staff.")
+   (min-base-key
+    :accessor min-base-key :initarg :min-base-key :initform nil
```

Comment:
Current code in `%create-new-target` assumes these are never nil; so maybe `:initform` could force them to be initialized?

```suggestion
    :accessor min-base-key :initarg :min-base-key :initform (error "min-base-key must be initialized")
```